_posts/2020/09/2020-09-30-release-notes.md

```diff
@@ -0,0 +1,42 @@
+---
+layout: post
+date: 2020-09-30
+title: "0.11: Release Notes"
+author: olenagerasimova
+description: |
+  Key features and improvements description in the recent
+  version 0.11, released just a few days ago.
```

Comment:
@olenagerasimova `0.11` was released, just now and is available on DockerHub: https://hub.docker.com/layers/artipie/artipie/0.11/images/sha256-d8f20f938c4ee2e3a0f4bd956538495ae48641bd0ae9a6974be469db4f16934a?context=explore

So we are ready to publish, I'd suggest to update `date` and remove "released just a few days ago" part